html-forms/index.html

```diff
@@ -18,7 +18,7 @@ <h1>Coding Form</h1>
       </select>
     </div>
     <div>
-      <p>Favorite Language (choose one)</p>
+      <label>Favorite Language? (choose one)</label><br />
       <input type="radio" name="language" value="HTML"/> HTML
       <input type="radio" name="language" value="CSS" /> CSS
       <input type="radio" name="language" value="Javascript" /> Javascript
```

Comment:
Each individual radio button will need its own label so that the user can click the text to choose the button

```html
<label>
  <input type="radio" name="language" value="HTML"/> HTML
</label>
```